src/app/cube/content-pages/contribute-page/contribute-page.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { sections } from './copy';
+
+@Component({
+  selector: 'clark-contribute-page',
+  templateUrl: './contribute-page.component.html',
+  styleUrls: ['./contribute-page.component.scss']
+})
+
+export class ContributePageComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void { }
+
+  title = 'Contribute';
+  copy = sections;
+  get tabs() {
+    return Object.values(this.copy);
```

Comment:
This is redundant to instantiate copy.
```suggestion
    return Object.values(sections);
```

Reply:
Removed the redundancy and replaced it with `return Object.values(sections);`